packages/client/components/NewAzureIssueInput.tsx

```diff
@@ -0,0 +1,249 @@
+  paddingLeft: 16,
+  width: '100%'
+})
+
+const NewIssueInput = styled('input')({
+  appearance: 'none',
+  background: 'transparent',
+  border: 'none',
+  color: PALETTE.SLATE_700,
+  fontSize: 16,
+  margin: 0,
+  padding: '0px 8px 0px 0px',
+  outline: 0,
+  width: '100%'
+})
+
+const Error = styled(StyledError)({
+  fontSize: 13,
+  textAlign: 'left',
+  width: '100%'
+})
+
+interface Props {
+  isEditing: boolean
+  meetingId: string
+  setIsEditing: (isEditing: boolean) => void
+  viewerRef: NewAzureIssueInput_viewer$key
+}
+
+const validateIssue = (issue: string) => {
+  return new Legitity(issue).trim().min(2, `C’mon, you call that an issue?`)
+}
+
+const NewAzureIssueInput = (props: Props) => {
+  const {isEditing, meetingId, setIsEditing, viewerRef} = props
+  const viewer = useFragment(
+    graphql`
+      fragment NewAzureIssueInput_viewer on User {
+        id
+        team(teamId: $teamId) {
+          id
+        }
+        teamMember(teamId: $teamId) {
+          integrations {
+            azureDevOps {
+              projects {
+                ...NewAzureIssueMenu_AzureDevOpsRemoteProjects
+                id
+                name
+                instanceId
+              }
+            }
+          }
+        }
+      }
+    `,
+    viewerRef
+  )
+  const {id: userId, team, teamMember} = viewer
+  const {id: teamId} = team!
+  const projects = teamMember?.integrations?.azureDevOps.projects || []
+  const atmosphere = useAtmosphere()
+  const {onCompleted, onError} = useMutationProps()
+  const [selectedProjectName, setSelectedProjectName] = useState(projects[0]?.name || '')
+  const {fields, onChange, validateField, setDirtyField} = useForm({
+    newIssue: {
+      getDefault: () => '',
+      validate: validateIssue
+    }
+  })
+  const {originRef, menuPortal, menuProps, togglePortal, portalStatus} = useMenu(
+    MenuPosition.UPPER_LEFT,
+    {isDropdown: true}
+  )
+  const ref = useRef<HTMLInputElement>(null)
+  const {dirty, error} = fields.newIssue
+  useEffect(() => {
+    if (portalStatus === PortalStatus.Exited) {
+      ref.current && ref.current.focus()
+    }
+  }, [portalStatus])
+
+  const handleCreateNewIssue = (e: FormEvent) => {
+    e.preventDefault()
+    if (portalStatus !== PortalStatus.Exited || !selectedProjectName) return
+    const {newIssue: newIssueRes} = validateField()
+    const {value: newIssueTitle, error} = newIssueRes
+    if (error) {
+      setDirtyField()
+      return
+    }
+    setIsEditing(false)
+    fields.newIssue.resetValue()
+    if (!newIssueTitle.length) {
+      fields.newIssue.dirty = false
+      return
+    }
+    const selectedProject = projects.find((project) => project.name === selectedProjectName)!
+    const serviceProjectHash = AzureDevOpsProjectId.join(
+      selectedProject.instanceId,
+      selectedProject.id
+    )
+    const newTask = {
+      teamId,
+      userId,
+      meetingId,
+      content: convertToTaskContent(`${newIssueTitle} #archived`),
```

Comment:
Is it expected behaviour? We always creating archived tasks in our board? Do you know the reason behind this?

Reply:
By creating an archived task, the `integration` is stored in the `task` and can be used to insert it into the Relay connection:
https://github.com/ParabolInc/parabol/blob/8e96045b57741440e39cd19cd8d38883fcbc629b/packages/client/mutations/handlers/handleGitHubCreateIssue.ts#L11

Also, as we already had this logic built for the task integration, I'm guessing it was easier to use the existing logic in `CreateTaskMutation` than create a new mutation (or update `UpdatePokerScopeMutation`) that adds the integration into the connection.

Does that sound right, @mattkrick?

Reply:
working as intended? yes
correct? no :disappointed: 
the problem was folks would run a poker meeting, import a bunch of tasks & then when they finish the meeting, those tasks would be on their kanban board. they didn't like this, and I can't blame them, they pointed "stories" but in our system, everything is a "Task". Then again, some users like having stories show up on their task board! How do we make both sides happy?
If someone would like to tackle this problem, it'd give me great joy

Reply:
I see - so the original reason to create a task when creating an integration issue was to point to the story? Why do we need to point to a story when we're querying directly from the API? If we create a GitHub issue from a Poker meeting, we can query that issue directly from GitHub's API without creating a Parabol task?